include/glow/Partitioner/Partitioner.h

```diff
@@ -38,6 +38,10 @@ struct BackendInfo {
   uint64_t memSize;
   /// Backend pointer.
   Backend *backend = nullptr;
+  /// The non-supported nodes kind.
+  std::set<Kinded::Kind> nonSupportedNodesKind;
```

Comment:
You store multiple kinds in the set, right? So, should it be called `nonSupportedNodesKinds`? Same for `supportedNodesKinds`?